riscv/sim.h

```diff
@@ -51,8 +52,11 @@ class sim_t : public htif_t, public simif_t
   }
   const char* get_dts() { if (dts.empty()) reset(); return dts.c_str(); }
   processor_t* get_core(size_t i) { return procs.at(i); }
+  const cfg_t &get_cfg() { return *cfg; }
   unsigned nprocs() const { return procs.size(); }
 
+  const std::map<size_t, processor_t*>& get_harts() { return harts; }
```

Comment:
2f5112f9dd94e0870f37711f3d99df4bfa6461d0 does far more than its commit message claims

Reply:
Rebase screwup.  Most of that stuff should be part of the final commit.